frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -127,6 +128,17 @@ const getAvailableFilters = (initialFilters) => {
   return filters;
 };
 
+const getLabelByKind = (kind) => {
+  const labels = {
+    ImageStream: 'Source to Image',
+    Template: 'Template',
+    InstalledOperator: 'Installed Operators',
+    ServiceClass: 'Service Clasee',
+  };
+
+  return labels[kind];
+};
```

Comment:
The values aren't the same strings noted here:
https://github.com/christianvogt/console/blob/204be559477989ec2b0d3501fcf04d4721d115af/frontend/public/components/catalog/catalog-items.jsx#L102-L125

Reply:
Used ```getAvailableFilters``` to get the label.